common/app/layout/Sublink.scala

```diff
@@ -352,12 +352,15 @@ case class PaidCard(
   image: Option[ImageMedia],
   fallbackImageUrl: Option[String],
   targetUrl: String,
+  cardTypes: Option[ItemClasses] = None,
   branding: Option[Branding]
 ) extends FaciaCard
 
 object PaidCard {
 
-  def fromPressedContent(content: PressedContent): PaidCard = {
+  def fromPressedContent(content: PressedContent): PaidCard = this.fromPressedContent(content, None)
```

Comment:
Explicitly calling the mapping function with the first param would make the compiler happy :D
```
val cards = collection.curatedPlusBackfillDeduplicated.map(PaidCard.fromPressedContent(_))
```